---a/net/minecraft/world/chunk/EmptyChunk.java
+++b/net/minecraft/world/chunk/EmptyChunk.java
@@ -1,9 +1,7 @@
 package net.minecraft.world.chunk;
 
-import com.google.common.base.Predicate;
 import java.util.List;
 import java.util.Random;
+
 import net.minecraft.block.Block;
 import net.minecraft.entity.Entity;
 import net.minecraft.init.Blocks;
@@ -11,110 +11,104 @@
 import net.minecraft.util.BlockPos;
 import net.minecraft.world.EnumSkyBlock;
 import net.minecraft.world.World;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class EmptyChunk extends Chunk
-{
+import com.google.common.base.Predicate;
+
+public class EmptyChunk extends Chunk {
+
     private static final String __OBFID = "CL_00000372";
 
-    public EmptyChunk(World worldIn, int x, int z)
-    {
+    public EmptyChunk(World worldIn, int x, int z) {
         super(worldIn, x, z);
     }
 
-    public boolean isAtLocation(int x, int z)
-    {
+    public boolean isAtLocation(int x, int z) {
         return x == this.xPosition && z == this.zPosition;
     }
 
-    public int getHeight(int x, int z)
-    {
+    public int getHeight(int x, int z) {
         return 0;
     }
 
-    public void generateSkylightMap() {}
+    public void generateSkylightMap() {
+    }
 
-    @SideOnly(Side.CLIENT)
-    public void generateHeightMap() {}
-
-    public Block getBlock(BlockPos pos)
-    {
+    public Block getBlock(BlockPos pos) {
         return Blocks.air;
     }
 
-    public int getBlockLightOpacity(BlockPos pos)
-    {
+    public int getBlockLightOpacity(BlockPos pos) {
         return 255;
     }
 
-    public int getBlockMetadata(BlockPos pos)
-    {
+    public int getBlockMetadata(BlockPos pos) {
         return 0;
     }
 
-    public int getLightFor(EnumSkyBlock p_177413_1_, BlockPos pos)
-    {
+    public int getLightFor(EnumSkyBlock p_177413_1_, BlockPos pos) {
         return p_177413_1_.defaultLightValue;
     }
 
-    public void setLightFor(EnumSkyBlock p_177431_1_, BlockPos pos, int value) {}
+    public void setLightFor(EnumSkyBlock p_177431_1_, BlockPos pos, int value) {
+    }
 
-    public int setLight(BlockPos pos, int p_177443_2_)
-    {
+    public int setLight(BlockPos pos, int p_177443_2_) {
         return 0;
     }
 
-    public void addEntity(Entity entityIn) {}
+    public void addEntity(Entity entityIn) {
+    }
 
-    public void removeEntity(Entity p_76622_1_) {}
+    public void removeEntity(Entity p_76622_1_) {
+    }
 
-    public void removeEntityAtIndex(Entity p_76608_1_, int p_76608_2_) {}
+    public void removeEntityAtIndex(Entity p_76608_1_, int p_76608_2_) {
+    }
 
-    public boolean canSeeSky(BlockPos pos)
-    {
+    public boolean canSeeSky(BlockPos pos) {
         return false;
     }
 
-    public TileEntity getTileEntity(BlockPos pos, Chunk.EnumCreateEntityType p_177424_2_)
-    {
+    public TileEntity getTileEntity(BlockPos pos, Chunk.EnumCreateEntityType p_177424_2_) {
         return null;
     }
 
-    public void addTileEntity(TileEntity tileEntityIn) {}
+    public void addTileEntity(TileEntity tileEntityIn) {
+    }
 
-    public void addTileEntity(BlockPos pos, TileEntity tileEntityIn) {}
+    public void addTileEntity(BlockPos pos, TileEntity tileEntityIn) {
+    }
 
-    public void removeTileEntity(BlockPos pos) {}
+    public void removeTileEntity(BlockPos pos) {
+    }
 
-    public void onChunkLoad() {}
+    public void onChunkLoad() {
+    }
 
-    public void onChunkUnload() {}
+    public void onChunkUnload() {
+    }
 
-    public void setChunkModified() {}
+    public void setChunkModified() {
+    }
 
-    public void getEntitiesWithinAABBForEntity(Entity entityIn, AxisAlignedBB aabb, List listToFill, Predicate p_177414_4_) {}
+    public void getEntitiesWithinAABBForEntity(Entity entityIn, AxisAlignedBB aabb, List listToFill, Predicate p_177414_4_) {
+    }
 
-    public void getEntitiesOfTypeWithinAAAB(Class entityClass, AxisAlignedBB aabb, List listToFill, Predicate p_177430_4_) {}
+    public void getEntitiesOfTypeWithinAAAB(Class entityClass, AxisAlignedBB aabb, List listToFill, Predicate p_177430_4_) {
+    }
 
-    public boolean needsSaving(boolean p_76601_1_)
-    {
+    public boolean needsSaving(boolean p_76601_1_) {
         return false;
     }
 
-    public Random getRandomWithSeed(long seed)
-    {
-        return new Random(this.getWorld().getSeed() + (long)(this.xPosition * this.xPosition * 4987142) + (long)(this.xPosition * 5947611) + (long)(this.zPosition * this.zPosition) * 4392871L + (long)(this.zPosition * 389711) ^ seed);
+    public Random getRandomWithSeed(long seed) {
+        return new Random(this.getWorld().getSeed() + (long) (this.xPosition * this.xPosition * 4987142) + (long) (this.xPosition * 5947611) + (long) (this.zPosition * this.zPosition) * 4392871L + (long) (this.zPosition * 389711) ^ seed);
     }
 
-    public boolean isEmpty()
-    {
+    public boolean isEmpty() {
         return true;
     }
 
-    public boolean getAreLevelsEmpty(int startY, int endY)
-    {
+    public boolean getAreLevelsEmpty(int startY, int endY) {
         return true;
     }
-}
+}
